### ISSUE TITLE
Weight decay and train data preprocessing

### ISSUE THREAD

**Issue by revilokeb** [marvis/pytorch-mobilenet#1]
@marvis Thanks for providing the benchmark numbers on mobilenet training! I am currently also training mobilenet, but I am not yet as far in the process. I was wondering about your main.py / run.sh and if you have run scripts using weight decay = 1e-4 (default)? The paper https://arxiv.org/pdf/1704.04861.pdf (p.4) seems to indicate that "it was important to put very little or no weight decay (l2 regularization) on the depthwise filters".
Additionally, the paper is stating that "[we] reduce the amount image of distortions by limiting the size of small crops that are used in large Inception training". However, http://pytorch.org/docs/torchvision/transforms.html#torchvision.transforms.RandomSizedCrop seems to be preprocessing used for large inception-type networks.
Have you used this during your training? Thx!

**Comment by marvis** [comment]
@revilokeb Thanks for your feedback! 
I trained the model exactly by the command "nohup python main.py -a mobilenet ImageNet-Folder > log.txt &". I only changed the max epoches to 120 without modify weight decay. BTW, the best performance is achieved after 114epoches.

**Comment by revilokeb** [comment]
@marvis Ok, many thanks for your reply!
